flink-end-to-end-tests/flink-elasticsearch1-test/src/main/java/org/apache/flink/elasticsearch1/test/Elasticsearch1SinkExample.java

```diff
@@ -15,10 +15,11 @@
  * limitations under the License.
  */
 
-package org.apache.flink.streaming.connectors.elasticsearch.examples;
+package org.apache.flink.elasticsearch1.test;
```

Comment:
I think the package should be:
`org.apache.flink.streaming.tests`

This would then be consistent with out other end-to-end test jobs

Reply:
Will change.